### DISCUSSION TITLE
Keypoints Extraction

### DISCUSSION THREAD

**Discussion by zohaibmaqsoodahmad** [ultralytics/ultralytics#4817] [Q&A]
Hi!
I want to extract  keypoints from an image which may contain 1 human or multiple human. But I only want keypoints. I don't want to display the image. just want to extract keypoints.

**Comment by glenn-jocher** [comment]
@zohaibmaqsoodahmad hello!

To extract keypoints for human pose estimation using YOLOv8, you'll want to use the model that's trained for pose/keypoint estimation tasks. Once you have the model and your dataset ready, you can run the predict mode to get the keypoints for each human detected in the image.

The output will typically include the coordinates for each keypoint. You can then process these coordinates as needed without displaying the image. Just make sure to set up your code to save or handle the keypoints data instead of visualizing it.

For detailed guidance on how to set up and run the predict mode for keypoint extraction, please refer to the Pose/Keypoint Estimation section in our documentation. 😊🚀

Happy coding!
